systemtest/src/test/java/io/strimzi/systemtest/UserST.java

```diff
@@ -171,12 +171,19 @@ void testUserWithQuotas() {
         assertThat(result.out().contains("producer_byte_rate=" + prodRate), is(true));
         assertThat(result.out().contains("consumer_byte_rate=" + consRate), is(true));
 
+        String zkListCommand = "sh /opt/kafka/bin/zookeeper-shell.sh localhost:21810 <<< 'ls /config/users'";
```

Comment:
Stupid question, but do we test these things also in ITs? Assuming we have ITs for this with a real Zoo and Kafka (I know there are some open issues). That would help to test this already at Travis.

Reply:
AFAIK no. I can do it.